bin/cij_excavator

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""
+    CIJOE Metric Excavator
+"""
+import argparse
+import importlib.util
+import sys
+import os
+import cij.util
+import cij
+
+def trun_tcase_fpaths(root):
+    """Yields absolute paths to testcases below the given 'root'"""
+
+    for path, _, _ in os.walk(root):
+        if path.endswith(".sh"):
+            yield path
+
+def main(args):
+    """
+    Apply the user-provided extractors to the given test-result output
+    """
+
+    args.extractors = []
```

Comment:
What is the intention of adding this to `args` instead of a plain variable?

Reply:
The ``args`` is CLI-arguments parsed by ``argparse``, this just extends it with a list of extractor-instances.